analyses/org.eclipse.tracecompass.incubator.executioncomparision.ui/src/org/eclipse/tracecompass/incubator/internal/executioncomparison/ui/ExecutionComparisonView.java

```diff
@@ -0,0 +1,1212 @@
+ * traces)
+ *
+ * @author Fateme Faraji Daneshgar
+ */
+@SuppressWarnings("restriction")
+public class ExecutionComparisonView extends DifferentialFlameGraphView implements ICheckboxTreeViewerListener {
+
+    /**
+     * the id of the view
+     */
+    public static final String VIEW_ID = "org.eclipse.tracecompass.incubator.internal.executioncomparison.ui.execComparison"; //$NON-NLS-1$
+    /**
+     * Default weights for organizing the view
+     */
+    private static final int[] DEFAULT_WEIGHTS_ShowQuery = new int[] { 450, 290, 260 };
+    private static final int[] DEFAULT_WEIGHTS_HideQuery = new int[] { 450, 100, 450 };
+    private static final int[] DEFAULT_WEIGHTS_FILTERING_H = new int[] { 3, 9 };
+    private static final int[] DEFAULT_WEIGHTS_TimeInterval = new int[] { 240, 380, 380 };
+
+    /** A composite that allows us to add margins for part A */
+    private @Nullable TmfXYChartViewer fChartViewerA;
+    private @Nullable SashForm fXYViewerContainerA;
+    private @Nullable TmfViewer fTmfViewerA;
+    private @Nullable SashForm fSashFormLeftChildA;
+    private @Nullable SashForm fSashFormLeftChildB;
+    private @Nullable IContextService fContextService;
+    private @Nullable Action fConfigureStatisticAction;
+    private List<IContextActivation> fActiveContexts = new ArrayList<>();
+
+    /** A composite that allows us to add margins for part B */
+    private @Nullable TmfXYChartViewer fChartViewerB;
+    private @Nullable SashForm fXYViewerContainerB;
+    private @Nullable TmfViewer fTmfViewerB;
+    private List<String> fTraceListA = new ArrayList<>();
+    private List<String> fTraceListB = new ArrayList<>();
+    private String fTraceStr = "Trace(s): ";//$NON-NLS-1$
+    private String fStatisticStr = "Statistic: ";//$NON-NLS-1$
+
+    /**
+     * The title of the view
+     */
+    @SuppressWarnings("null")
+    public static final String VIEW_TITLE = Messages.multipleDensityViewtitle;
+    private ITmfTimestamp fStartTimeA = TmfTimestamp.BIG_BANG;
+    private ITmfTimestamp fStartTimeB = TmfTimestamp.BIG_BANG;
+    private ITmfTimestamp fEndTimeA = TmfTimestamp.BIG_CRUNCH;
+    private ITmfTimestamp fEndTimeB = TmfTimestamp.BIG_CRUNCH;
+    @SuppressWarnings("null")
+    private String fStatistic = Messages.multipleDensityViewDuration;
+    private JFormattedTextField ftextAFrom = new JFormattedTextField();
+    private JFormattedTextField ftextBFrom = new JFormattedTextField();
+    private JFormattedTextField ftextATo = new JFormattedTextField();
+    private JFormattedTextField ftextBTo = new JFormattedTextField();
+    private @Nullable Text ftextQuery;
+    private TmfTimestampFormat fFormat = new TmfTimestampFormat("yyyy-MM-dd HH:mm:ss.SSS.SSS.SSS"); //$NON-NLS-1$
+    private @Nullable Listener fSashDragListener;
+    private @Nullable SashForm fsashForm = null;
+    private static final String TMF_VIEW_UI_CONTEXT = "org.eclipse.tracecompass.tmf.ui.view.context"; //$NON-NLS-1$
```

Comment:
All the static strings and all the static attributes should be grouped together.